### DISCUSSION TITLE
Laravel 9.x and future roadmap?

### DISCUSSION THREAD

**Discussion by shanecp** [laravel/framework#34629] [General]
I've searched and couldn't find any information regarding the future roadmap of Laravel versions. Especially with 6 month release cycles, it becomes a bit difficult to keep up and update the packages when new major versions are released.

As Jeffrey Way shown while upgrading Laracast's own codebase (https://laracasts.com/series/whats-new-in-laravel-8/episodes/11), it slows down most developers with unnecessary having to fork dependencies and creating workarounds. It would be good to know the general roadmap of upcoming changes, and some transition period so that we can prepare and update the dependencies.

An example where this becomes a very practical issue is when we've used `laravel/ui` and it gets replaced by Jetstream with little notice. Because we're from an agency, that means we have to schedule time for training and I assume a lot of others will appreciate that as well.

**Comment by AlexWinder** [comment]
I'm not sure if there is an actual roadmap but you can view the files which have been changed using the GitHub difftool between specific branches.

https://github.com/laravel/laravel/compare/master...develop

But I agree, having 3 different authentication packages in the last few updates has been a bit of a headache, first there was `make:auth`, then `ui`, and now `fortify`. At least now with the likes of Fortify it is more front-end agnostic and so you're not bound to any particular front-end framework. But I agree constantly releasing new packages for what seems to be the sake of staying fresh is frustating at times. I would much prefer that the core functionality of things like authentication be made as good as they possibly can rather than creating a new package just so that there is something to announce at Laracon.

**Reply by shanecp** [reply]
Well, there's a new one now - https://github.com/laravel/breeze

**Reply by SuperDJ** [reply]
https://github.com/laravel/ideas/issues/2491

**Comment by jithinjose2** [comment]
https://laravelversions.com/

**Comment by alifhossain174** [comment]
https://laravel.com/docs/8.x/releases

**Comment by Gustavovaler** [comment]
Laravel is moving from a 6 month major release cycle to a 12 month release cycle.
[https://blog.laravel.com/updates-to-laravels-versioning-policy](url)

**Comment by stackdevelopers** [comment]
Watch for Laravel 9 updates:
https://youtu.be/bT25AowPjBM
